visualizer/src/Project/Load.js

```diff
@@ -6,22 +6,25 @@ import ProjectContext from './ProjectContext';
 import createLoadMachine from './service/loadMachine';
 import createProjectMachine from './service/projectMachine';
 
-function Load({ id, track, spots }) {
+function Load({ id, spots }) {
```

Comment:
Potentially a naive question, but why is spots still getting passed in here but track is not?

Reply:
Good catch, spots is not actually passed to the component and should be removed here